model-optimizer/extensions/ops/range.py

```diff
@@ -69,4 +74,8 @@ def infer(node: Node):
 
         assert start is not None and limit is not None and delta is not None, \
             'Range operation {} with dynamic inputs is not supported'.format(name)
-        node.out_port(0).data.set_value(np.arange(start, limit, delta, dtype=node.soft_get('dtype', start.dtype)))
+
+        if not node.has_valid('output_type'):
+            node['output_type'] = start.dtype
```

Comment:
We should not set the `output_type` attribute for the Range-1. Otherwise the MO IR reader will generate a different model. So, please, rewrite the shape infer function so it works correctly for both v1 and v4 Range.

Reply:
And *may be* `type_infer` should be updated as well.

Reply:
It won't get into IR if backend_attrs will not have it.